.github/workflows/release-snapshot.yml

```diff
@@ -0,0 +1,36 @@
+name: Release Snapshot to Sonatype
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release_snapshot_sonatype:
+    if: "github.event.release.prerelease"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: corretto
+          java-version: 8
+          cache: sbt
+      - name: Release Snapshot to Sonatype
+        run: |
+          VERSION=$(git describe --tags | cut -f2 -d"@")
+          if [[ ${VERSION:0:1} == "v" ]] ; then
+          VERSION=${VERSION:1}
+          fi
+          if [[ ${VERSION: -9} != "-SNAPSHOT" ]] ; then
+            echo "Version must end in -SNAPSHOT. Adding -SNAPSHOT suffix"
+            VERSION="$VERSION-SNAPSHOT"
+          fi
+          echo $PGP_SECRET | base64 --decode | gpg --batch --import
+          export GPG_TTY=$(tty)
+          echo "Releasing version $VERSION Sonatype as snapshot"
+          yes | sbt -DRELEASE_TYPE=snapshot "clean" "project client" "release release-version $VERSION with-defaults"
```

Comment:
This pipes the output of `yes` into the sbt commands - this means we can integrate with the existing build projects (which require manual input).

Note that we also need to define the release type as a system prop to integrate with the current build steps too.

Reply:
I think this is only releasing for:

content-api-client_2.12
content-api-client_2.13

Maybe we'll need to run `release-cross` to release the default client at the same time?

Reply:
We created a CAPI key and added it to the repository secrets. We also removed `"project client"` from the `sbt` command, so the GitHub Action now releases both `content-api-client-default` and `content-api-client`